src/main/java/com/github/checkstyle/generatepatchfile/GeneratePatchFile.java

```diff
@@ -277,6 +263,25 @@ private void generateDiffPatchWithGitCommand(int headNum, String patchFormat) th
         }
     }
 
+    private void runShellCommand(String command) throws Exception {
+        final List<String> showCmds = new ArrayList<>();
+        showCmds.add("sh");
+        showCmds.add("-c");
+        showCmds.add(command);
+        final Process process = new ProcessBuilder()
+                .directory(new File(repoPath))
+                .command(showCmds)
+                .start();
+        final int code = process.waitFor();
+        stateException(code, "an error occurred when running" + command);
```

Comment:
please put a space at the end of the string:
`"an error occurred when running "`

Reply:
done